pom.xml

```diff
@@ -29,10 +29,10 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-actuator</artifactId>
 		</dependency>
-        <dependency>
-            <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-starter-data-jpa</artifactId>
-        </dependency>
+		<dependency>
```

Comment:
I am not sure what do you mean by hard tabs. I am using eclipse and I generally format the code using the shortcode (Ctrll+Shift+F). Is that wrong practice for pom.xml?